spyder/plugins/plots/widgets/figurebrowser.py

```diff
@@ -145,6 +147,11 @@ def setup_toolbar(self):
                 tip=_("Save All Images..."),
                 triggered=self.thumbnails_sb.save_all_figures_as)
 
+        copyfig_btn = create_toolbutton(
+            self, icon=ima.icon('editcopy'),
+            tip=_("Copy Image (%s)" % get_shortcut('plots', 'copy')),
```

Comment:
On thinking about it further, since this is a tooltip we could probably be more descriptive here, e.g. something like ``Copy plot to clipboard as image``.